tests/sync_pipfile_test.py

```diff
@@ -54,3 +54,27 @@ def test_sync_dev_pipfile_no_original(tmp_path):
         text = setup_file.read_text()
         assert "pytest~=5.1" in text, text
         assert "requirementslib~=1.5" in text, text
+
+
+@pytest.mark.parametrize(
+    ("source_pipfile_dirname", "update_count"),
+    [("dash_or_underscore_0", 23)],
+)
+def test_sync_underscore_or_dash(
+    capsys, tmp_path, shared_datadir, source_pipfile_dirname, update_count
+):
+    """
+    sync --pipfile should work for either dash or underscore names.
+
+    Asserts fix for https://github.com/Madoshakalaka/pipenv-setup/issues/72.
+    """
+    pipfile_dir = shared_datadir / source_pipfile_dirname
```

Comment:
This suggestion is paired with a couple additional ones

```suggestion
    pipfile_dir = shared_datadir / "dash_or_underscore_0"
```